docs/docsite/rst/conf.py

```diff
@@ -227,7 +227,7 @@
 html_context = {
     'display_github': 'True',
     'show_sphinx': False,
-    'is_eol': True,
+    'is_eol': False,
```

Comment:
How about
```suggestion
    'is_eol': False if if tags.has('core_lang') else True,
```
?

This would be equivalent to
```suggestion
    'is_eol': not tags.has('core_lang'),
```
but I think the above is easier to understand - at least for me :)